source/admin_manual/list_of_events.rst

```diff
@@ -1013,6 +1013,21 @@ Mailbox
 |                     | .. versionadded:: v2.3.10                            |
 +---------------------+------------------------------------------------------+
 
+.. _event_mail_expunged:
+
+mail_expunged
+^^^^^^^^^^^^^
+
+.. versionadded:: 2.3.16
+
+A mail was expunged from the mailbox.
```

Comment:
Maybe add a clarification: Note that this event inherits from mailbox, not mail.

Reply:
fixed